WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/EditableOrderViewModelTests.swift

```diff
@@ -486,6 +486,24 @@ final class EditableOrderViewModelTests: XCTestCase {
         XCTAssertTrue(viewModel.paymentDataViewModel.supportsAddingCouponToOrder)
     }
 
+    // MARK: - Add Products to Order via SKU Scanner Tests
+
+    func test_add_product_to_order_via_sku_scanner_feature_flag_is_enabled_then_feature_support_returns_true() {
```

Comment:
Non blocker here: to add _when_ to the test function name:

_test_add_product_to_order_via_sku_scanner_when_the_feature_flag_is_enabled_then_feature_support_returns_true_